README.md

```diff
@@ -13,24 +13,47 @@ rather than this. This component **does not** polyfill interfaces like `VRFrameD
 
 ## How It Works
 
-`CardboardVRDisplay` uses DeviceMotionEvents to implement a complementary
-filter which does [sensor fusion and pose prediction][fusion] to provide
-orientation tracking. It can also render in stereo mode, and includes mesh-based
+As of 1.0.4, `CardboardVRDisplay` uses [RelativeOrientationSensor] for orientation tracking,
+falling back to [DeviceMotionEvents] using [sensor fusion and pose preidction][fusion].
+It can also render in stereo mode, and includes mesh-based
 lens distortion. This display also includes user interface elements in VR mode
 to make the VR experience more intuitive, including:
 
-- A gear icon to select your VR viewer.
-- A back button to exit VR mode.
-- An interstitial which only appears in portrait orientation, requesting you switch
+* A gear icon to select your VR viewer.
+* A back button to exit VR mode.
+* An interstitial which only appears in portrait orientation, requesting you switch
   into landscape orientation (if [orientation lock][ol] is not available).
 
-[fusion]: http://smus.com/sensor-fusion-prediction-webvr/
-[ol]: https://www.w3.org/TR/screen-orientation/
+### iframes
+
+[RelativeOrientationSensor] is a new API ([read more about the new Sensors on the web][sensors])
+first implemented in Chrome M63. This API uses the new [Feature Policy] specification which allows
+developers to selectively enable or disable browser features. By default, main frames and
+same-origin iframes have access to Sensor APIs, but cross-origin iframes must specify
+feature policy and allow `gyroscope` and `accelerometer` features. If your experience
+is attempting to use the native [WebXR Device API] in an iframe, you'll have to specify
+that feature as well ([WebXR's feature name may change](https://github.com/immersive-web/webxr/issues/308)). All of these features require HTTPS to function, except for `localhost`, where HTTP
+is allowed.
+
+```html
+<iframe src="https://otherdomain.com" allow="gyroscope; accelerometer; xr" />
+```
+
+While devicemotion is a fallback for Sensors, eventually devicemotion will be behind the same
```

Comment:
`devicemotion` is backticked [below](https://github.com/immersive-web/cardboard-vr-display/pull/14/files#diff-04c6e90faac2675aa89e2176d2eec7d8R50) but not here